relayer-cli/src/components.rs

```diff
@@ -36,13 +37,14 @@ impl JsonTracing {
     #[allow(trivial_casts)]
     pub fn new(cfg: GlobalConfig) -> Result<Self, FrameworkError> {
         let filter = build_tracing_filter(cfg.log_level);
-        let use_color = false;
 
         // Construct a tracing subscriber with the supplied filter and enable reloading.
         let builder = FmtSubscriber::builder()
+            .with_target(false)
             .with_env_filter(filter)
             .with_writer(std::io::stderr as StdWriter)
-            .with_ansi(use_color)
+            // Note: JSON output is currently un-affected by ANSI 'color' option.
+            .with_ansi(COLORED_OUTPUT)
```

Comment:
Can we just set it to `false` here, since it's no affected anyway?

Reply:
Indeed, I wasn't sure what to do about it.

It makes sense to make it `false`. The alternative is to remove the `.with_ansi(COLORED_OUTPUT)` build option altogether. I'll set it to false.